packages/components/src/styles/components/app-frame.scss

```diff
@@ -16,32 +16,47 @@
   grid-template-rows: auto 1fr auto;
   grid-template-columns: auto 1fr;
   min-height: 100vh;
+}
+
+// OPTIONS
 
-  // prevent interaction with AppHeader when SideNav is open/expanded
-  &:has(.hds-side-nav--is-not-minimized) .hds-app-header--is-mobile {
-    pointer-events: none;
+// Make header position sticky/fixed if is-header-fixed is set and the viewport height is greater than 480px
+@media (height >= 480px) {
+  .hds-app-frame--is-header-fixed {
+    .hds-app-frame__header {
+      position: sticky;
+      top: 0;
+      right: 0;
+      left: 0;
+    }
+  
+    // Modify sidenav layout when used together with fixed app-header
+    .hds-app-frame__sidebar {
+      top: var(--token-app-header-height);
+      height: calc(100vh - var(--token-app-header-height));
```

Comment:
I have a couple of thoughts:

Should we be more opinionated on this? I know various consumers have brought up this being something they might want control over, but this component isn't used in the same way as others. This probably will be set and likely not changed.

Is the request from consumers because they _actually_ want to change this interaction (make the `AppHeader` not sticky)? If our guidance is to have both nav elements sticky, do they intend to go against this (and do they have a good enough reason to)?

To me, this request seems a little bit like consumers want the appearance of control over this interaction pattern, but don't actually intend to go against our guidance (and if they are, they we should be asking why)

Reply:
Currently the SideNav is fixed/sticky and that's how it's behaved from its initial release and I haven't known of complaints about that behavior (although I may just be ignorant of complaints.)

The consensus so far from designers seems to be to make the AppHeader fixed/sticky and not give consumers a built in option to override this behavior for sake of consistency.

I tend to agree that these type of cross-product standard navigation components should behave as consistently as practical.

So I'm going to leave all this alone awhile longer to see if there's any further discussion... (And then I will remove the option to "unfix" the header if that's the true consensus.)

Reply:
For future reference (copy&pasting&rewording from a thread in Slack)

> Deciding that the AppFrame, which is a very generic, high-level/page-level layout component, has by default a fixed header and sidebar, I mean engrained in the code implementation, instead of being an argument/prop, is an antipattern. We can easily guide our consumers to set it to true (or do it ourselves, since it’s likely in the consumers codebase there is [a very limited number of instances](https://github.com/search?q=%3CHds%3A%3AAppFrame+user%3Ahashicorp+repo%3Ahashicorp%2Fcloud-ui&type=code&ref=advsearch)).
> [Pre-assigning sticky behaviour to the Header/Sidebar in the AppFrame] is contrast with how we’ve implemented most of our components (in code) in our design system. Components should be as agnostic as possible, be black boxes that know only (ideally) about themselves, and not about their parents or their children. Their interface with the inside/outside world is via their API.

That said, my suggestion is to escalate the decision to the Core Squad Lead(s).

Reply:
From discussion, I've gone ahead and removed the `isHeaderFixed` argument along with related code